code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -213,6 +213,40 @@
 	new /obj/item/surgicaldrill(src)
 	new /obj/item/circular_saw(src)
 
+/obj/item/storage/firstaid/ert
+	name = "ert first-aid kit"
+	icon_state = "bezerk"
+	desc = "A medical kit used by Nanotrasen emergency response team personnel."
+
+/obj/item/storage/firstaid/ert/populate_contents()
+	new /obj/item/healthanalyzer/advanced(src)
+	new /obj/item/stack/medical/splint(src)
+	new /obj/item/reagent_containers/applicator/dual(src)
+	new /obj/item/stack/medical/ointment/advanced(src)
+	new /obj/item/stack/medical/bruise_pack/advanced(src)
+	new /obj/item/storage/pill_bottle/ert(src)
+	new /obj/item/storage/pill_bottle/patch_pack/ert/(src)
+
+/obj/item/storage/firstaid/ert/empty/populate_contents()
+	return
```

Comment:
Is this part of the code necessary? I don't think the empty variant of the ERT specific medkit would be used anywhere.

Reply:
Ditto for the amber

Reply:
It probably would not be necessary, but I was following similar kit builds within the code. The only thing I could think of its use is if an admin spawned in an empty kit to do some type of custom inventory population given for an event or specific situation.

Reply:
Removed the empties. Tested to see if the kits, pill bottles, and patch packs still function. Resolved with [e4e67ed](https://github.com/ParadiseSS13/Paradise/pull/20561/commits/e4e67ed1e4c82c11e5f8893ded02a79acea61102)